src/Symfony/Component/Validator/Mapping/Deprecated/TraversalStrategy.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Mapping\Deprecated;
+
+trigger_error('Constants STOP_RECURSION in class Symfony\Component\Validator\Mapping\TraversalStrategy is deprecated since version 2.3 and will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
This constant is used in 3 files:

 * `src/Symfony/Component/Validator/Mapping/GenericMetadata.php:148`
 * `src/Symfony/Component/Validator/Mapping/MemberMetadata.php:229`
 * `src/Symfony/Component/Validator/Validator/RecursiveContextualValidator.php:370,609,637,725,743`